docs/build/sdk/indexs.md

```diff
@@ -33,106 +33,69 @@ the minimum amount.
 ```typescript
 const sdk = await SDK.initialize(endpoint);
 
-const poolId = await sdk.models.createCpmmMarketAndDeployAssets(
+const res = await sdk.models.createCpmmMarketAndDeployAssets({
   signer,
   oracle,
-  marketPeriod,
-  advised ? "Advised" : "Permissionless",
+  period: marketPeriod,
   marketType,
   mdm,
-  amts,
-  baseAssetAmount,
-  wts,
-  kp,
+  amount,
+  weights: weights.split(`,`),
   metadata,
-  false
-);
+  callbackOrPaymentInfo: false
+});
 ```
 
-**Arguments** | Name | Type | Introduction | | ---- | ---- | ------------ | |
-signer | KeyringPairOrExtSigner | The actual signer provider to sign the
-transaction. | | oracle | string |The address that will be responsible for
-reporting the market. | | period | MarketPeriod |Start and end block numbers or
-unix timestamp of the market. | | creationType | string |"Permissionless" or
-"Advised", Advised as default | | marketType | string |"Categorical" or "Scalar"
-| | mdm | MarketDisputeMechanism |Dispute settlement can be authorized, court or
-simple_disputes | | keep | string[] |Specifies how many assets to keep. | |
-weights | string[] |List of relative denormalized weights of each asset price. |
-| baseAssetAmount | Amount for native currency liquidity | | amounts | string[]
-| List of amounts of each outcome asset that should be deployed.| | metadata |
-DecodedMarketMetadata |Market metadata | | paymentInfo | |"true" to get txn fee
-estimation otherwise "false" |
+**Object Arguments** 
```

Comment:
```suggestion
**Object Arguments**

```

Otherwise, the table doesn't render correctly.

Reply:
There's some other issue which is causing table rendering. @Whisker17 is working on it, I think.